README.md

```diff
@@ -2,3 +2,71 @@
 
 Instructions and Dockerfiles for tracking flights with your Raspberry Pi and a USB TV stick.
 
+Inspired by Alex Ellis' blog post: [Get eyes in the sky with your Raspberry Pi](https://blog.alexellis.io/track-flights-with-rpi/)
+
+This repo contains the following:
+
+* dump1090 - A docker container for [MalcolmRobb/dump1090](https://github.com/malcolmrobb/dump1090) that is a simple Mode S decoder for [RTLSDR devices](http://flightaware.com/adsb/prostick/).
+* flightaware - A docker container for FlightAware.com's [PiAware](http://flightaware.com/adsb/piaware/) feeder app.
+* docker-compose.yml - A docker compose file where all configuration is made.
+
+## Simple setup
+
+1. Make sure Docker is installed on your raspberry pi:
+
+    ```
+    $ curl -sSL https://get.docker.com | sh
+    ```
+
+2. Clone the repo on your raspberry pi:
+
+    ```
+    $ git clone https://github.com/LoungeFlyZ/eyes-in-the-sky
+    ```
+
+3. Download a copy of docker-compose for raspberry pi:
+
+    ```
+    $ wget -O docker-compose https://github.com/javabean/arm-compose/releases/download/1.16.1/docker-compose-Linux-armv7l
```

Comment:
Get this via official route through pip

Reply:
FlightAware moved to not using MAC address anymore.  They have a "feeder id" (in the compose yml file) now. Once up and running you can grab the feeder id you have been assigned and update your compose file.

Regarding compose installation.  I couldnt find how to get a more recent compose build for arm via pip.  any pointers?